yarn-project/pxe/src/contract_function_simulator/pxe_oracle_interface.ts

```diff
@@ -388,20 +389,20 @@ export class PXEOracleInterface implements ExecutionDataProvider {
 
     const contractName = await this.contractDataProvider.getDebugContractName(contractAddress);
     if (currentIndex !== oldIndex) {
-      await this.taggingDataProvider.setTaggingSecretsIndexesAsSender(
-        [new IndexedTaggingSecret(appTaggingSecret, currentIndex)],
-        sender,
-      );
+      await this.taggingDataProvider.setTaggingSecretIndexAsSender(directionalAppTaggingSecret, currentIndex);
 
-      this.log.debug(`Syncing logs for sender ${sender} at contract ${contractName}(${contractAddress})`, {
-        sender,
-        secret: appTaggingSecret,
-        index: currentIndex,
-        contractName,
-        contractAddress,
-      });
+      this.log.debug(
+        `Syncing logs for secret ${directionalAppTaggingSecret.toString()} at contract ${contractName}(${contractAddress})`,
+        {
+          index: currentIndex,
+          contractName,
+          contractAddress,
+        },
+      );
     } else {
-      this.log.debug(`No new logs found for sender ${sender} at contract ${contractName}(${contractAddress})`);
+      this.log.debug(
+        `No new logs found for secret ${directionalAppTaggingSecret.toString()} at contract ${contractName}(${contractAddress})`,
+      );
```

Comment:
These logs now became worse but I print out enough info in PrivateExecutionOracle that still makes it possible to associate the directional tagging secret with sender.